src/open_inwoner/templates/pages/faq.html

```diff
@@ -2,7 +2,7 @@
 {% load i18n faq_tags %}
 
 {% block content %}
-    <h1 class="h1">
+    <h1 class="utrecht-heading-1">
```

Comment:
Gemeente specific class :D

Reply:
@Bartvaderkin Actually it's not quite :-D this is one of the NL Design-System (design token) classes we will be using more and more often - the first municipality that sets the standard, also gives the code their name... Since open-inwoner is not a municipality we will not be suggesting names for classes etc. In future you will see more Utrecht en Den Haag names even :-O

Reply:
Ugh, well ok then. Let's hope they thought about how this is going to scale in the future :shrug:

Reply:
Expectation is that it will take at least 5 years for all of these names to become something like "NLDS-heading-bla-bla" once there will be definite community standards 🙈